chutney/ui/src/app/modules/campaign/components/execution/history/campaign-executions-history.component.ts

```diff
@@ -100,9 +100,10 @@ export class CampaignExecutionsHistoryComponent implements OnInit, OnDestroy {
 
     private replay() {
         const lastReport = this.campaignReports[0]
-        this.campaignService.executeCampaign(this.campaign.id, lastReport.report.executionEnvironment, lastReport.report.dataset).pipe(
-            map(campaignExecutionReport => this.refreshCampaign())
-        ).subscribe()
+        this.campaignService.executeCampaign(this.campaign.id, lastReport.report.executionEnvironment, lastReport.report.dataset).subscribe()
```

Comment:
No need to broadcast event because the it (the event) is listened in the same component
`this.onExecuteSubscription = this.eventManagerService.subscribe('execute', () => this.refreshCampaign());`
You can use `refreshCampaign` directly